site/pt/beta/tutorials/keras/basic_regression.ipynb

```diff
@@ -0,0 +1,865 @@
+{
```

Comment:
<p>Shouldn't this be translated too?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>